.github/workflows/sync-docs-full.yml

```
@@ -0,0 +1,87 @@
+          
+          # Debug - show all found paths
+          echo "Found these MDX files:"
+          echo "$ALL_PATHS"
+          
+          # Create JSON arrays for both changed and removed
+          # We want to remove and re-add everything to get a fresh start
+          CHANGED_JSON="["
+          REMOVED_JSON="["
+          FIRST_CHANGED=true
+          FIRST_REMOVED=true
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)